### ISSUE TITLE
[EP1] Typos for scripts in ep1_citadel_04 and ep1_c17_00

### ISSUE THREAD

**Issue by BreakinBenny** [ValveSoftware/Source-1-Games#3747]
When Alyx EMPs the Combine door (door_rollertraining) shut, the **trigger_player_closedoor** entity tries to make it play the **idle_close** animation instead of **idle_closed**. The former is incorrect. Also, below I've included the fixed ep1_c17_00.pat file contents:

`"Entities"`
{
	"replace_entity"
	{
	"origin" "4930.31 -5905.47 -108"
	"targetname" "lcs_duckdown"
	"SceneFile" "scenes/episode_1/c17/al_tunnel_01a.vcd"
	"busyactor" "1"
	"classname" "logic_choreographed_scene"
	"hammerid" "1843024"
	"OnCompletion" "al_c17_00_fieldhack,Start,,0,-1"
	"OnTrigger1" "ss_alyx_walllean,BeginSequence,,0,-1"
	}
}`

**Comment by Yetoo1** [comment]
If the .pat file is unrelated to the issue presented in the beginning make these separate issues. What has been fixed in the .pat file?

**Comment by BreakinBenny** [comment]
> If the .pat file is unrelated to the issue presented in the beginning make these separate issues. What has been fixed in the .pat file?

You're right. The point entity tries to play the scene named "al_tunnel_01a .vcd", but the existing file is without a space at the end and this stops Alyx from emoting before scene **al_tunnel_01.vcd** plays.

```
{
"origin" "4930.31 -5905.47 -108"
"SceneFile" "scenes/episode_1/c17/al_tunnel_01a .vcd"
"targetname" "lcs_duckdown"
"busyactor" "1"
"classname" "logic_choreographed_scene"
"OnCompletion" "al_c17_00_fieldhack,Start,,0,-1"
"OnTrigger1" "ss_alyx_walllean,BeginSequence,,0,-1"
}
```

I've tried to replace the trigger_once entity with one that has the correct output, but I failed to do so because of something I don't know about... Maybe because I didn't have the hammerid for that brush.

**Comment by Yetoo1** [comment]
@BreakinBenny I mean to open two issues on this issue tracker. Modify this for idle_close/idle_closed issue and make another issue for idle_close/idle_closed or vice versa.

**Comment by BreakinBenny** [comment]
No problem Yetoo1, I split them.
